sdk/storage/AzureStorageBlob/Source/StorageBlobClient.swift

```diff
@@ -344,41 +339,111 @@ public class StorageBlobClient: PipelineClient, PagedCollectionDelegate {
         }
     }
 
-    /**
-     Download a blob from a given URL.
-     - Parameter url: A URL to a blob to download.
-     - Parameter options: A `DownloadBlobOptions` object to control the download operation.
-     - Parameter completion: An `HTTPResultHandler` closure that returns a `BlobStreamDownloader` object on success.
-     */
-    public func download(
+    /// Download a blob from a given URL.
+    /// - Parameters:
+    ///   - url: A URL to a blob to download.
+    ///   - options: A `DownloadBlobOptions` object to control the download operation.
+    ///   - completion: An `HTTPResultHandler` closure that returns a `BlobStreamDownloader` object on success.
+    public func rawDownload(
```

Comment:
We'll want to think about the names of these method pairs, and also make sure we make the same naming choices in Android.

Reply:
Agreed. This was just a temporary rename because I didn't want to throw away a working implementation.